### PR TITLE
Add response time metric

### PR DESCRIPTION
**Pull request by p-blomberg**

### CODE REVIEW

File: cloudping.py [hunk 1]
```diff
@@ -27,6 +28,7 @@ def ping(event, context):
             allow_redirects=options['allow_redirects'],
             timeout=options['timeout'])
         response.raise_for_status()
+        response_time = response.elapsed.total_seconds()
```
Comment (edruid) [inline cloudping.py:31]: Should we set response_time in the except block as well?

File: cloudping.py [hunk 2]
```diff
@@ -38,17 +40,13 @@ def ping(event, context):
...
 
     client = boto3.client('cloudwatch')
-    response = client.put_metric_data(
-        Namespace='cloudping',
-        MetricData=[
-            {
-                'MetricName': 'status',
-                'Dimensions': [
+    dimensions = [
                     {
```
Comment (edruid) [inline cloudping.py]: Please fix the indentation level for dimensions

**Review by edruid — CHANGES_REQUESTED** [review]

File: cloudping.py (hunk 2 above)
Comment (p-blomberg) [reply cloudping.py]: I have no idea what the correct indentation level would be.

File: cloudping.py (hunk 1 above)
Comment (p-blomberg) [reply cloudping.py:31]: I don't see a reason to do that. If the request fails, the response time is not relevant.

**Review by edruid — APPROVED** [review]

**Pull request merged**